dt-posts/custom-post-type.php

```diff
@@ -76,8 +76,9 @@ public function register_post_type(){
             'show_in_rest'          => false
         ];
 
+        // Adjust defaults accordingly, prior to registration.
+        $defaults = apply_filters( 'dt_register_post_type_defaults', $defaults );
```

Comment:
@kodinkat pass in the $post_type so, all only change the values on the ppl groups post type